res/css/views/rooms/_EventTile.scss

```diff
@@ -298,13 +298,13 @@ limitations under the License.
     cursor: pointer;
 }
 
-.mx_EventTile_e2eIcon[hidden] {
+.mx_EventTile_e2eIcon.hidden {
```

Comment:
All CSS classes must exist in the `mx_` namespace. I would suggest `mx_EventTile_e2eIcon_hidden`.

We do this to avoid dependencies that define CSS colliding with ours (i.e. another library may have declared a `hidden` class).